src/frontend/sharedComponents/Editor/PresetView.tsx

```diff
@@ -31,7 +33,9 @@ export const PresetView = ({
         {PresetIcon}
         <Text style={styles.categoryName}>{presetName}</Text>
       </View>
-      <Text style={styles.changeButtonText}>{formatMessage(m.change)}</Text>
+      {!isTrack && (
+        <Text style={styles.changeButtonText}>{formatMessage(m.change)}</Text>
+      )}
```

Comment:
Ok I passed a disabled prop and used it. I didn't use it for the other props? Not sure what you meant by that since you said we want to display the Change text.

Reply:
was referring to having a single prop to determine the functionality of the touchable here. so instead of having both `onPressPreset` and `disabled` determining how the it behaves and looks. my assumption is that we should only show the "Change" text if the button can actually be pressed, so the end result i was imagining was something along the lines of:

```tsx
    <TouchableOpacity 
      disabled={disabled} // basically just updating this line
      onPress={onPressPreset} // could technically update this line to be like `disabled ? onPressPreset : undefined`, but not strictly necessary in this case
      style={styles.preset}>
      <View style={{flexDirection: 'row', alignItems: 'center'}}>
        {PresetIcon}
        <Text style={styles.categoryName}>{presetName}</Text>
      </View>
      {!disabled && <Text style={styles.changeButtonText}>{formatMessage(m.change)}</Text>}
    </TouchableOpacity>
```